src/sentry/integrations/gitlab/issues.py

```diff
@@ -39,6 +39,7 @@ def get_create_issue_config(self, group, **kwargs):
                 'url': autocomplete_url,
                 'updatesForm': True,
                 'required': True,
+                'choices': [],
```

Comment:
This fixes a warning from React.proptypes visible in the browser console.

Reply:
Very small thing, but might've been nice to put this in a separate one-liner PR that we could have mashed approve on.

Reply:
Good point. :+1: